packages/backend/src/managers/playgroundV2Manager.ts

```diff
@@ -101,19 +92,19 @@ export class PlaygroundV2Manager extends Publisher<PlaygroundV2[]> implements Di
     systemPrompt: string | undefined,
     trackingId: string,
   ): Promise<string> {
-    const id = `${this.#playgroundCounter++}`;
-
     if (!name) {
-      name = this.getFreeName();
+      name = `playground ${this.getUniqueId()}`;
```

Comment:
This does not ensure unicity, as the user can set the name itself with a value that would be the same as a future unique Id.

Reply:
I restore the getFreeName method ✅